server/datastore/mysql/statistics.go

```diff
@@ -82,6 +87,7 @@ func (d *Datastore) ShouldSendStatistics(ctx context.Context, frequency time.Dur
 		NumUsers:                  amountUsers,
 		NumTeams:                  amountTeams,
 		NumPolicies:               amountPolicies,
+		NumLabels:                 amountLabels,
 		SoftwareInventoryEnabled:  appConfig.HostSettings.EnableSoftwareInventory,
 		VulnDetectionEnabled:      &appConfig.VulnerabilitySettings.DatabasesPath != nil,
```

Comment:
While we're at it, change to:
`VulnDetectionEnabled:      appConfig.VulnerabilitySettings.DatabasesPath != "",`
(address of a variable will always be non-nil)

(I ran `staticcheck` locally a couple of days ago.)